recipes/sparc-x-api/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "SPARC-X-API" %}
+{% set version = "1.0.0" %}
+{% set file_ext = ".tar.gz" %}
+
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/SPARC-X/{{ name }}/archive/refs/tags/v{{ version }}{{ file_ext }}
+  sha256: 34aa3a8d0801a91736b0d8322d78d1e432e92585e61af54db3aba17340ff7617
+
+build:
+  noarch: python
+  number: 0
+  # Install and include the psp files
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv && cd .. && {{ PYTHON }} -m sparc.download_data"
+  entry_points:
+    - "sparc-ase = sparc.cli:main"
+    
+
+requirements:
+  build:
+    - python >=3.8
+    - setuptools
+    - ase >=3.22
+    - pip
+  host:
+    - python >=3.8
+    - ase >=3.22
+    - pip
+  run:
+    - python >=3.8
+    - ase >=3.22
+
+test:
+  imports:
+    - sparc
+    - sparc.io
+  requires:
+    - pip
+  commands:
+    - pip check
+    - python -m sparc.quicktest
+    - sparc-ase help
+
+about:
+  home: https://github.com/SPARC-X/{{ name }}
+  license: GPL-3.0-or-later
+  license_family: GPL
+  license_file: LICENSE
+  summary: "SPARC-X-API: A Python API for the SPARC-X DFT Code"
+  description: |
+    `SPARC-X-API` is an [ASE](https://wiki.fysik.dtu.dk/ase/)-compatible Python API for the density functional theory (DFT) code [SPARC](https://github.com/SPARC-X/SPARC). It offers:
+    1. ASE-compatible I/O format for SPARC files
+    2. A JSON API interfacing with SPARC's C-code for parameter validation and conversion
+    3. A comprehensive calculator interface for SPARC.
+  doc_url: https://github.com/SPARC-X/{{ name }}/blob/master/README.md
+  dev_url: https://github.com/SPARC-X/{{ name }}
```

The jinja variable here obfuscates the URLs and makes them unclickable.

```suggestion
  home: https://github.com/SPARC-X/SPARC-X-API
  license: GPL-3.0-or-later
  license_family: GPL
  license_file: LICENSE
  summary: "SPARC-X-API: A Python API for the SPARC-X DFT Code"
  description: |
    `SPARC-X-API` is an [ASE](https://wiki.fysik.dtu.dk/ase/)-compatible Python API for the density functional theory (DFT) code [SPARC](https://github.com/SPARC-X/SPARC). It offers:
    1. ASE-compatible I/O format for SPARC files
    2. A JSON API interfacing with SPARC's C-code for parameter validation and conversion
    3. A comprehensive calculator interface for SPARC.
  doc_url: https://github.com/SPARC-X/SPARC-X-API/blob/master/README.md
  dev_url: https://github.com/SPARC-X/SPARC-X-API
```